skins/oasis/css/core/ads.scss

```diff
@@ -47,6 +48,26 @@ $taboola-small-breakpoint: "#{$screen} and (max-width: 420px)";
 	}
 }
 
+.bfaa-template {
+	.WikiaTopAds {
+		margin-top: 0;
+		padding: 0 0 $global-navigation-height 0;
+		width: 100%;
+
+		&.bfaa-pinned-nav {
+			padding: 0;
+		}
+
+		.WikiaTopAdsInner {
+			width: 100%;
+
+			#TOP_LEADERBOARD {
```

Comment:
an id selector that is 4 levels nested. 
Please try to refactor this selectors as this will get compiled to: `.bfaa-template .WikiaTopAds .WikiaTopAdsInner #TOP_LEADERBOARD` that is not a selector I would like to write by hand.

Try to keep to nesting 2, max 3 levels deep